fe1-web/parts/lao/organizer/eventCreation/events/CreateElection.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import {
+  View, Platform, TextInput, ScrollView,
+} from 'react-native';
+import 'react-datepicker/dist/react-datepicker.css';
+import { useNavigation } from '@react-navigation/native';
+
+import STRINGS from 'res/strings';
+import DatePicker from 'components/DatePicker';
+import ParagraphBlock from 'components/ParagraphBlock';
+import WideButtonView from 'components/WideButtonView';
+import { Timestamp } from 'model/objects';
+import TextBlock from 'components/TextBlock';
+import DropdownSelector from 'components/DropdownSelector';
+
+function dateToTimestamp(date: Date): Timestamp {
+  return new Timestamp(Math.floor(date.getTime() / 1000));
+}
+
+/**
+ * UI to create a Election Event
+ *
+ * TODO Send the Election event to the organization server when the confirm button is pressed
+ */
+const CreateElection = ({ route }: any) => {
+  const styles = route.params;
+  const navigation = useNavigation();
+  const initialStartDate = new Date();
+  const initialEndDate = new Date();
+  // Sets initial end date to 1 hour later than start date
+  initialEndDate.setHours(initialEndDate.getHours() + 1);
+
+  const [startDate, setStartDate] = useState(dateToTimestamp(initialStartDate));
+  const [endDate, setEndDate] = useState(dateToTimestamp(initialEndDate));
+
+  const [electionName, setElectionName] = useState('');
+  const [electionQuestion, setElectionQuestion] = useState('');
+  const [electionBallots, setElectionBallots] = useState(['']);
+  const [electionBallotCounter, setElectionBallotCounter] = useState(2);
```

Comment:
You are using the magic number "2" a few times. Consider using a constant representing the minimum amount of ballot options

Reply:
This is implemented